dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -163,7 +163,7 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId, userI
 function shareProject() {
   dashboard.project.save(function () {
     var origin = location.protocol + '//' + location.host;
-    var shareUrl = origin + dashboard.project.appToProjectUrl() + '#' + dashboard.project.getCurrentId();
+    var shareUrl = origin + dashboard.project.appToProjectUrl() + '/' + dashboard.project.getCurrentId();
```

Comment:
Would it make sense to move the logic for formatting the project id parameter into a helper function like appToProjectUrl()?  We would then need to update only one place for any future URL format changes.